frontend/src/metabase/visualizations/visualizations/Progress/Progress.jsx

```diff
@@ -248,8 +248,8 @@ export default class Progress extends Component {
             )}
           </div>
           <div className={CS.mt1}>
-            <span className="float-left">0</span>
-            <span className="float-right">{t`Goal ${formatValue(
+            <span className={CS.floatLeft}>0</span>
+            <span className={CS.floatRight}>{t`Goal ${formatValue(
```

Comment:
![image](https://github.com/metabase/metabase/assets/1937582/f360111c-19a7-440e-b27c-5b6fb2495e06)
![image](https://github.com/metabase/metabase/assets/1937582/a17a2f83-af93-4aa8-8674-7a561ace0738)

These classes are used in progress chart

To create a progress chart. Orders > Count by row > visualize > select progress chart > configure the goal